docs/api-bridges.md

```diff
@@ -6,7 +6,7 @@ title: Bridges
 To make use of any schema, uniforms have to create a _bridge_ of it - a unified schema mapper.
 
 <p align="center">
-  <img src="/img/bridge-concept.svg" />
+  <img src="/img/bridge-concept.svg" alt=''/>
```

Comment:
This shouldn't pass Prettier. Can you check why it's not working here? If it's not able to check that, then at least revise it manually: double quotes, space before the tag end, and maximum line width.